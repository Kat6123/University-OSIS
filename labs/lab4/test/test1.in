hi
there
i'm Katya
ouuuuuu
wooo
fooo
