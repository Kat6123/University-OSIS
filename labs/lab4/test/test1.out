ih
ereht
aytaK m'i
uuuuuuo
ooow
ooof
